test/Formik.test.tsx

```diff
@@ -1296,4 +1297,34 @@ describe('<Formik>', () => {
     // so it should change
     expect(injected.isValidating).toBe(false);
   });
+
+  it('should merge validation errors', async () => {
+    const validate = () => ({
+      users: [{ firstName: 'required' }],
+    });
+    const validationSchema = Yup.object({
+      users: Yup.array().of(
+        Yup.object({
+          lastName: Yup.mixed().required('required'),
+        })
+      ),
+    });
+    let injected: any;
+
+    const tree = shallow(
```

Comment:
We don’t need enzyme here. Can just use ReactDOM.render() on an empty node like we do in Field